TapCounter/ViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ViewController.swift
+//  TapCounter
+//
+//  Created by Alice Aponasko on 1/24/16.
+//  Copyright © 2016 justforfun. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```

Comment:
Could you leave inline comments like so? So I can see what line you are referring to.